simple-todo/package.json

```diff
@@ -4,22 +4,23 @@
   "author": "Paul Adams",
   "private": true,
   "scripts": {
-    "dev": "webpack-dev-server --inline --hot",
-    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
+    "dev": "webpack-dev-server --mode=development --inline --hot",
+    "build": "webpack --mode=production --progress --hide-modules"
   },
   "dependencies": {
```

Comment:
It is overkill to detail the dependency versions in the manner that I have. Just wanted to be explicit as a sanity check.